src/main/java/edu/greenblitz/robotName/subsystems/swerve/Modules/mk4iSwerveModule/MK4ISwerveModule.java

```diff
@@ -1,75 +1,66 @@
 
 import com.ctre.phoenix.motorcontrol.ControlMode;
 import com.ctre.phoenix.motorcontrol.DemandType;
 import com.ctre.phoenix.motorcontrol.NeutralMode;
 import com.ctre.phoenix.motorcontrol.TalonFXControlMode;
-import com.ctre.phoenixpro.hardware.CANcoder;
+import com.ctre.phoenix.sensors.CANCoder;
 import edu.greenblitz.robotName.subsystems.Battery;
 import edu.greenblitz.robotName.subsystems.swerve.Chassis.SwerveChassis;
+import edu.greenblitz.robotName.subsystems.swerve.Modules.ISwerveModule;
+import edu.greenblitz.robotName.subsystems.swerve.Modules.SwerveModuleInputsAutoLogged;
 import edu.greenblitz.robotName.subsystems.swerve.SwerveModuleConfigObject;
-import edu.greenblitz.robotName.subsystems.swerve.constants.MK4iSwerveConstants;
 import edu.greenblitz.robotName.utils.Conversions;
 import edu.greenblitz.robotName.utils.motors.GBFalcon;
 import edu.wpi.first.math.controller.SimpleMotorFeedforward;
+import edu.wpi.first.math.geometry.Rotation2d;
 import edu.wpi.first.math.util.Units;
 
 public class MK4ISwerveModule implements ISwerveModule {
 
-    private GBFalcon angularMotor;
-    private GBFalcon linearMotor;
-    private CANcoder canCoder;
-    private SimpleMotorFeedforward feedforward;
-    private double encoderOffset;
+    private final GBFalcon angularMotor;
+    private final GBFalcon linearMotor;
+    private final CANCoder canCoder;
+    private final SimpleMotorFeedforward linearFeedForward;
+    private final double encoderOffset;
 
 
     public MK4ISwerveModule(SwerveChassis.Module module) {
 
-        SwerveModuleConfigObject configObject;
-        switch (module) {
-
-            case FRONT_LEFT:
-                configObject = MK4iSwerveConstants.SdsModuleFrontLeft;
-                break;
-            case FRONT_RIGHT:
-                configObject = MK4iSwerveConstants.SdsModuleFrontRight;
-                break;
-            case BACK_LEFT:
-                configObject = MK4iSwerveConstants.SdsModuleBackLeft;
-                break;
-            case BACK_RIGHT:
-                configObject = MK4iSwerveConstants.SdsModuleBackRight;
-                break;
-            default:
-                throw new IllegalArgumentException("Invalid module");
-        }
+        SwerveModuleConfigObject configObject = switch (module) {
+            case FRONT_LEFT -> MK4iSwerveConstants.MK4I_MODULE_FRONT_LEFT;
+            case FRONT_RIGHT -> MK4iSwerveConstants.MK4I_MODULE_FRONT_RIGHT;
+            case BACK_LEFT -> MK4iSwerveConstants.MK4I_MODULE_BACK_LEFT;
+            case BACK_RIGHT -> MK4iSwerveConstants.MK4I_MODULE_BAK_RIGHT;
+            default -> throw new IllegalArgumentException("Invalid module");
+        };
 
         angularMotor = new GBFalcon(configObject.angleMotorID);
-        angularMotor.config(new GBFalcon.FalconConfObject(MK4iSwerveConstants.baseAngConfObj));
+        angularMotor.config(new GBFalcon.FalconConfObject(MK4iSwerveConstants.ANGULAR_FALCON_CONFIG_OBJECT));
```

Comment:
Out of scope but `FalconConfObject`?

Consider cleaning up in the future

Reply:
in the next branch (converting the mk4i swerve to phoenix 6) ctre has a configuration object built in for their motors so i'll use it in the near future